### PR TITLE
enable proxy post submission via url param

### PR DESCRIPTION
**Pull request by danfrankj**
By going to 

`posteditor?proxy=true`

Users can add proxy posts 

image

@NiharikaRay @matthewwardrop @earthmancash

### CODE REVIEW

**Comment by danfrankj** [comment]
Also as a quick heads up - I don't have time to fully clean up the webeditor code so if this is a step in the right direction let's do it rather than going for fully clean code at this point.

File: knowledge_repo/app/templates/post_editor_markdown.html [hunk 1]
```diff
@@ -0,0 +1,234 @@
+{% extends "post_editor_base.html" %}
```
Comment (NiharikaRay) [inline knowledge_repo/app/templates/post_editor_markdown.html]: just for my own understanding, what you did with the `html` files is to break it out from 1 template into 4 different ones yes?

**Comment by danfrankj** [comment]
@earthmancash @matthewwardrop last call for comments - reaching out since this is a larger change

**Comment by matthewwardrop** [comment]
This looks fine to me @danfrankj . 👍

**Comment by earthmancash** [comment]
this lgtm! Will be exciting seeing the proxy posts come in

**Pull request merged**